pkg/agent/baker_test.go

```diff
@@ -475,8 +475,23 @@ var _ = Describe("Assert generated customData and cseCmd", func() {
 					TransparentHugePageDefrag:  "defer+madvise",
 					SwapFileSizeMB:             &swapFileSizeMB,
 				}
-			}, nil),
-
+			}, func(o *nodeBootstrappingOutput) {
+				Expect(o.vars["KUBELET_CONFIG_FILE_ENABLED"]).NotTo(BeEmpty())
```

Comment:
nit: the empty check seems redundant if we check for equality immediately after

Reply:
yep, fixed